cpp/src/parser/environment.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <memory>
+#include <string_view>
+#include <unordered_map>
+
+#include "value.hpp"
+
+struct Environment;
+using EnvironmentP = std::shared_ptr<Environment>;
+
+struct Environment : public std::enable_shared_from_this<Environment>
+{
+	EnvironmentP parent{};
```

Comment:
The majority of the new code you wrote uses an 8 space indent as opposed to this 2 space indent. Please change these 2 space indents (or vice-versa) to be consistent with the rest of the code.

Reply:
no, it doesn't. you're just seeing a github PR bug.